tests/integration/_conftest/protostar_project_fixture.py

```diff
@@ -98,6 +99,22 @@ def create_contracts(self, contract_name_to_file: Dict[str, Union[str, Path]]):
             self._mocker, self._project_root_path
         )
 
+    def create_contracts_cairo1(
+        self, contract_name_to_contract_dir: Dict[str, Union[str, Path]]
+    ):
+        for contract_name, contract_dir in contract_name_to_contract_dir.items():
+            assert Path(
+                contract_dir
+            ).is_dir(), "contracts in cairo1 should be represented as directories"
+            relative_path = f"src/{contract_name}"
+            shutil.copytree(
+                str(contract_dir), str(self._project_root_path / relative_path)
```

Comment:
Are casts to string necessary here? I think it works with plain Path.

Reply:
yes, we can go with paths